pkgs/development/tools/vend/default.nix

```diff
@@ -1,28 +1,36 @@
 { stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoModule {
+buildGoModule rec {
   pname = "vend";
-  version = "unstable-2020-06-04";
 
-  patches = [./remove_tidy.patch];
+  /*
+  This package is used to generate vendor folders for
+  packages that use the `runVend` option with `buildGoModule`.
+
+  Do not update this package without checking that the vendorSha256
+  hashes of packages using the `runVend` option are unchanged
+  or updating their vendorSha256 hashes if necessary.
+  */
+  version = "1.0.2";
+  # Disable the bot
+  # nixpkgs-update: no auto update
+
+  # Disable `mod tidy`, patch was refused upstream
```

Comment:
can you add a link to the PR/issue here?

Reply:
done